testing/vcs/test_vcs_set_line_attribute_from_line.py

```diff
@@ -6,16 +6,22 @@
 l.type = "dash"
 
 v=vcs.createvector()
-v.line = "vcs_test_set_line"
+v.setLineAttributes("vcs_test_set_line")
```

Comment:
Sets linetype, linecolor and linewidth from the line with the name 'vcs_test_set_line'.